jwst/cube_build/ifu_cube.py

```diff
@@ -843,6 +845,7 @@ def build_ifucube_single(self):
                     self.spaxel_var = self.spaxel_var + np.asarray(spaxel_var, np.float64)
                     self.spaxel_iflux = self.spaxel_iflux + np.asarray(spaxel_iflux, np.float64)
                     result = None
+                    del result, spaxel_flux, spaxel_var, spaxek_iflux
```

Comment:
This is what's killing all the regression tests: there's a typo in "spaxek_iflux" (last argument in the list). Should be "spaxel_iflux".

Reply:
Surprised the pep8/codestyle checks didn't find that.